### ISSUE TITLE
queryParameters not marked as Deprecated

### ISSUE THREAD

**Issue by GrandLarseny** [BottleRocketStudios/iOS-Hyperspace#59]
After upgrading to Hyperspace 2.0.0, it looks like the old query parameters are no longer utilized. I'm assuming the intended use is now to return a URL with the query parameters already appended. This is of course acceptable, but it would be best to have a compiler warning to let the developer know that this value does not work as expected.

The following code no longer appends the querystring parameter `type` to a URL, for instance. 

```
    var queryParameters: [URLQueryItem]? {
        if let type = type {
            return [URLQueryItem(name: "type", value: type.rawValue)]
        }

        return nil
    }
```

**Comment by wmcginty** [comment]
Good catch. Will take a look at an appropriate deprecation warning.

**Comment by wmcginty** [comment]
Fixed by #61 . Thanks @GrandLarseny !
